src/App.vue

```diff
@@ -26,11 +26,25 @@
       ></ErrorAlertComponent>
 
     </div>
+    <!-- Global Snackbar -->
+    <v-snackbar
```

Comment:
Global snackbar, as any snackbars embedded into components don't trigger if you also redirect out of that route after an action